src/Mapping/Validator/SymfonyValidator.php

```diff
@@ -14,9 +15,13 @@ class SymfonyValidator implements IEntityValidator
 	/** @var Reader */
 	private $reader;
 
-	public function __construct(Reader $reader)
+	/** @var ConstraintValidatorFactoryInterface|null */
+	private $constraintValidatorFactory;
+
+	public function __construct(Reader $reader, ?ConstraintValidatorFactoryInterface $constraintValidatorFactory = null)
```

Comment:
I like it, can you please change it to setter?

Reply:
Setter is not a problem, but i do not know, how to correctly write it i neon,, i am trying somenting like this

```yaml
api:
    plugins:
        Apitte\Core\DI\Plugin\CoreMappingPlugin:
            request:
                validator:
                    factory: Apitte\Core\Mapping\Validator\SymfonyValidator(@phpdoc.reader)
                    setup:
                        - setConstraintValidatorFactory(Contributte\Validator\ContainerConstraintValidatorFactory())
```
And i am getting `Nette\InvalidArgumentException: Argument is not valid Statement entity.`

Reply:
It must be defined as service and then passed to validator?

```yaml
api:
    plugins:
        Apitte\Core\DI\Plugin\CoreMappingPlugin:
            request:
                validator: @Apitte\Core\Mapping\Validator\SymfonyValidator

services:
    -
        factory: Apitte\Core\Mapping\Validator\SymfonyValidator(@phpdoc.reader)
        setup:
            - setConstraintValidatorFactory(Contributte\Validator\ContainerConstraintValidatorFactory())
```